moshi/src/main/java/com/squareup/moshi/Moshi.java

```diff
@@ -184,7 +184,7 @@
   @CheckReturnValue
   public Moshi.Builder newBuilder() {
     Builder result = new Builder();
-    for (int i = 0, limit = lastOffset; i < limit; i++) {
+    for (int i = 0; i < lastOffset; i++) {
```

Comment:
This replaces a local read with a field read. Probably not a meaningful performance regression but in general we like optimized loops that don’t have to repeatedly read memory when they don’t have to.